CMakeLists.txt

```diff
@@ -171,9 +171,16 @@ endif()
 
 # need to add appropriate flags for stdexec
 set(CMAKE_CXX_FLAGS
-    "${CMAKE_CXX_FLAGS} -stdpar=${STDPAR} -mp=${OMP} --gcc-toolchain=/opt/cray/pe/gcc/12.2.0/bin/ -pthread"
+    "${CMAKE_CXX_FLAGS} -stdpar=${STDPAR} -mp=${OMP} -pthread"
```

Comment:
do we need -pthread flag here?

Reply:
removed. was there from before but no longer needed as things compile just fine without it.